cove_360/lib/threesixtygiving.py

```diff
@@ -285,13 +285,14 @@ def common_checks_360(context, upload_dir, json_data, schema_obj):
         'validation_errors_grouped': group_validation_errors(context['validation_errors'], context['file_type'], openpyxl_workbook)
     })
 
-    for test_classes_type in ['quality_accuracy', 'usefulness']:
+    for test_class_type in ['quality_accuracy', 'usefulness']:
         extra_checks = run_extra_checks(
-            json_data, cell_source_map, TEST_CLASSES[test_classes_type], ignore_errors=True, return_on_error=None)
+            json_data, cell_source_map, TEST_CLASSES[test_class_type], ignore_errors=True, return_on_error=None)
+
         context.update({
-            '{}_errored'.format(test_classes_type): extra_checks is None,
-            '{}_checks'.format(test_classes_type): extra_checks,
-            '{}_checks_count'.format(test_classes_type): (len(extra_checks) if extra_checks else 0) + (1 if context['data_only'] else 0)
+            '{}_errored'.format(test_class_type): extra_checks is None,
+            '{}_checks'.format(test_class_type): extra_checks,
+            '{}_checks_count'.format(test_class_type): (len(extra_checks) if extra_checks else 0) + (1 if context['data_only'] else 0)
         })
 
```

Comment:
Is this related to "quality checks in numbers" ? if not it is a good refactor that could be in a separate commit.

Reply:
You are right, this should had been in a different commit. I will leave it as it is now if you don't mind.